src/App.jsx

```diff
@@ -14,70 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const message = selectedGood
+    ? `${selectedGood} is selected`
+    : 'No goods selected';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+      <h1 className="title is-flex is-align-items-center">
+        {message}
+        {
+          selectedGood && (
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+              className="delete ml-3"
+              onClick={() => setSelectedGood(null)}
+            />
+          )
+        }
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {
+            goods.map((good) => {
+              const CURRENT_GOOD = selectedGood === good;
+              const SYMBOL_INSIDE_BUTTON = CURRENT_GOOD ? '-' : '+';
+              /* eslint-disable-next-line max-len */
+              const VALUE_INSIDE_DATA = CURRENT_GOOD ? 'RemoveButton' : 'AddButton';
```

Comment:
```suggestion
              const VALUE_INSIDE_DATA = CURRENT_GOOD
                ? 'RemoveButton' : 'AddButton';
```

remove eslint disable, write like this